Assets/Scripts/Lua/CLRBindings/Overworld/LuaGeneralOW.cs

```diff
@@ -31,7 +38,7 @@ public class LuaGeneralOW {
         }
         TextMessage[] textmsgs = new TextMessage[texts.Table.Length];
         for (int i = 0; i < texts.Table.Length; i++)
-            textmsgs[i] = new TextMessage(texts.Table.Get(i + 1).String, formatted, false, mugshots != null ? mugshots.Type == DataType.Table ? mugshots.Table.Get(i+1) : mugshots : null);
+            textmsgs[i] = new TextMessage(texts.Table.Get(i + 1).String, formatted, false, (mugshots != null && mugshots.Type == DataType.Table) ? mugshots.Table.Get(i+1) : null);
```

Comment:
The way it was done before allowed someone to be able to use the same "mugshot" in the entire text. Why did you remove it?